ci-operator/step-registry/gather/log-bundle-from-private-cluster/gather-log-bundle-from-private-cluster-commands.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -o nounset
+# set -o errexit
+# set -o pipefail
+
+set -x
+
+installer_bin=$(which openshift-install)
+echo "openshift-install binary path: $installer_bin"
+
+echo "openshift-install version:"
+openshift-install version
+
+function run_command() {
+    local cmd="$1"
+    echo "Running Command: ${cmd}"
+    eval "${cmd}"
+}
+
+function run_ssh_cmd() {
+    local sshkey=$1
+    local user=$2
+    local host=$3
+    local remote_cmd=$4
+    local options=$5
+    options+=" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o VerifyHostKeyDNS=yes "
+    cmd="ssh ${options} -i \"${sshkey}\" ${user}@${host} \"${remote_cmd}\""
+    run_command "$cmd" || return 2
+    return 0
+}
+
+function run_scp_to_remote() {
+    local sshkey=$1
+    local user=$2
+    local host=$3
+    local src=$4
+    local dest=$5
+    local options=$6
+    options+=" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o VerifyHostKeyDNS=yes "
+    cmd="scp ${options} -i \"${sshkey}\" ${src} ${user}@${host}:${dest}"
+    run_command "$cmd" || return 2
+    return 0
+}
+
+function run_scp_from_remote() {
+    local sshkey=$1
+    local user=$2
+    local host=$3
+    local src=$4
+    local dest=$5
+    local options=$6
+    options+=" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o VerifyHostKeyDNS=yes "
+    cmd="scp ${options} -i \"${sshkey}\" ${user}@${host}:${src} ${dest}"
+    run_command "$cmd" || return 2
+    return 0
+}
+
+ssh_key=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+bastion_dns=$(head -n 1 "${SHARED_DIR}/bastion_public_address")
+bastion_user=$(head -n 1 "${SHARED_DIR}/bastion_ssh_user")
+
+echo "Gathering log-bundle from private cluster"
+
+run_scp_to_remote "${ssh_key}" "${bastion_user}" "${bastion_dns}" "${installer_bin}" "/tmp/"
+run_ssh_cmd "${ssh_key}" "${bastion_user}" "${bastion_dns}" "/tmp/openshift-install gather bootstrap --dir /tmp/installer 2> /tmp/gather.log"
+run_scp_from_remote "${ssh_key}" "${bastion_user}" "${bastion_dns}" "/tmp/gather.log" "${ARTIFACT_DIR}/gather.log"
+log_bundle_path=$(tail -n 1 "${ARTIFACT_DIR}/gather.log"  | awk '{print $7}' | sed 's/"//g')
```

Comment:
How about directly run `ls *log-bundle*` command to get the file name instead of grep the file name in the log.

Reply:
updated.